minimum-constraints.txt

```diff
@@ -52,6 +52,7 @@ tabulate==0.8.2
 prompt-toolkit==1.0.15; python_version == '2.7'
 prompt-toolkit==2.0.1; python_version >= '3.4'
 PyYAML==5.1
+pydicti=1.1.3
```

Comment:
Change `=` to `==` to fix the build error.

Reply:
FIXED